src/main/cliapp/src/components/ontologies/CHEBIOntologyComponent.js

```diff
@@ -67,7 +67,7 @@ export const CHEBIOntologyComponent = () => {
   return (
       <div>
         <div className="card">
-            <h3>ZFA Table</h3>
+            <h3>CHEBI Table</h3>
```

Comment:
This should be ChEBI with a lower case 'h'

Reply:
Done